python/torch_mlir_e2e_test/test_suite/arange.py

```diff
@@ -248,3 +248,66 @@ def forward(self):
 @register_test_case(module_factory=lambda: ArangeFalsePinMemoryModule())
 def ArangeFalsePinMemoryModule_basic(module, tu: TestUtils):
     module.forward()
+
+
+# ==============================================================================
+
+class LinspaceModuleFloat32(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+    ])
+    def forward(self):
+        return torch.linspace(3, 10, 5, dtype=torch.float32)
```

Comment:
Use `torch.ops.aten.` in place of `torch.`. Do this for all the tests.

Reply:
Fixed